schemas/code_signature.yml

```diff
@@ -0,0 +1,59 @@
+---
+- name: code_signature
+  title: Code Signature
+  group: 2
+  description: These fields contain information about binary code signatures.
+  type: group
+  reusable:
+    top_level: false
+    expected:
+      - file
+      - process
+      # - dll
+      # - driver
+  fields:
+
+    - name: subject_name
+      format: string
+      level: core
+      type: keyword
+      description: Subject name of the code signer
+      example: Microsoft Corporation
+
+
+    - name: valid
+      level: core
+      type: boolean
+      description: Boolean to capture if the digital signature is verified against the binary content.
+      example: "true"
+
+
+    - name: exists
+      format: core
+      level: extended
```

Comment:
Remove the `format:` attribute.

Now the question is: was the intent to make this field level core or extended? :-)